src/ontology/cl-edit.owl

```diff
@@ -28247,6 +28248,19 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16230044"^^xsd:string) r
 AnnotationAssertion(rdfs:label obo:CL_4030025 "renal cortical fibroblast")
 EquivalentClasses(obo:CL_4030025 ObjectIntersectionOf(obo:CL_0000057 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005270)))
 
+# Class: obo:CL_4032000 (club-like cell of the urethral epithelium)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30566875"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:32497356"^^xsd:string) obo:IAO_0000115 obo:CL_4032000 "An epithelial cell of the urethra that has an expression profile similar to lung club cells. Club-like cells of the urethra epithelium are similar to lung club cells in their expression of  SCGB1A1 and in their enrichment of immunomodulatory programs.")
```

Comment:
There is an extra space before "SCGB1A1" in the text definition.

Reply:
hmmm we should probably implement a check for double space